dao/Entity.js

```diff
@@ -0,0 +1,26 @@
+const uuid = require('uuid')
+
+module.exports = require('fashion-model').extend({
+  properties: {
+    /**
+    * Id of the entity being stored in the database
+    */
+    entityId: String
+  },
+
+  init () {
+    if (!this.getEntityId()) {
+      this.setEntityId(uuid.v4())
+    }
+  },
+
+  prototype: {
+    getId () {
+      return this.getEntityId()
+    },
+
+    setId (id) {
+      this.setEntityId(id)
+    }
+  }
```

Comment:
Why not just use `id` as the property and doing this proxying? These methods seem pointless. Alternatively, just leave the `entityId` as is and use the built-in `fashion-model` getters.